modules/jaeger-rn-new-features.adoc

```diff
@@ -9,6 +9,20 @@ Result – If changed, describe the current user experience.
 ////
 
 [id="jaeger-rn-new-features_{context}"]
+== New features {ProductName} 1.24.0
+
+This release of {ProductName} includes the following new features:
+
+* Adds support for the Red Hat Elasticsearch Operator 5.0.
```

Comment:
@JStickler we have tested in ESO 5.0 and in 5.1

Reply:
Updated!